src/main/java/com/atlassian/bitbucket/jenkins/internal/client/HttpRequestExecutor.java

```diff
@@ -45,6 +47,24 @@ public interface HttpRequestExecutor {
      */
     <T> T executeGet(HttpUrl url, ResponseConsumer<T> consumer, RequestConfiguration... additionalConfig);
 
+    /**
+     * Executes a streaming get call to a given URL.
+     *
+     * @param url              The URL to hit on bitbucket server end
+     * @param additionalConfig additional configuration, such as credentials, that will be used in making calls
+     * @param consumer         on successful execution, {@link Response} will be passed to consumer
```

Comment:
`consumer` is unused, and should be removed I think. It doesn't quite make sense to both return an `InputStream` AND take in a `consumer`